lib/index.js

```diff
@@ -80,28 +86,29 @@ exports.balm = (opts = {}) => {
     console.log('Balm watching...');
 
     _socket = new WebSocket.Server({ port });
-    addHmrToBundles(port, blocksDir);
-
-    const watcher = 'chokidar';
-    const chokidar = require(watcher);
-    const distWatchOptions = {
-      awaitWriteFinish: {
-        stabilityThreshold: 100,
-        pollInterval: 25
-      }
-    };
-    const bundles = chokidar.watch(['./public/dist/*.css', './public/dist/*.js'], distWatchOptions);
-    const balmFiles = chokidar.watch(`${root}/**/*.balm`);
 
-    bundles.on('change', (path) => {
-      if (hasBundled[path]) {
+    fs.watch('./public/dist', (_, path) => {
+      if (!hasBundled[path]) {
+        console.log('File changed for HMR', path);
```

Comment:
We already have a console log in the notifyClient function. Does that not suffice?

Reply:
Done

Reply:
Up to you. Previously, we had removed a different version of this log, and we joked/semi-seriously talked about it being helpful and maybe I shouldn't have pushed to have it removed, so I re-added it. I can get rid of It again if you want.